src/services/liking.service.js

```diff
@@ -13,6 +13,19 @@ const LikingService = {
     }
     like.update({ isLiked: !like.isLiked });
     return sendResult(res, 200, ` The accommodation like  status updated to ${like.isLiked}`, like);
+  },
+
+  async getLikes(req, res) {
+    const { accommodationId } = req;
+    const countLikes = await db.Likings
+      .findAndCountAll({ where: { accommodationId, isLiked: true }, raw: false });
+    const countUnlikes = await db.Likings
+      .findAndCountAll({ where: { accommodationId, isLiked: false }, raw: false });
+    const data = {
+      likes: countLikes.count,
+      unlikes: countUnlikes.count
+    };
+    return sendResult(res, 200, ` The ccommodation ${accommodationId} has:  ${data.likes} likes, ${data.unlikes} unlikes `, data);
```

Comment:
Since we don't need the `req` or `res`, we only have to return the number of `likes`, then return them from the controller.
- Alternatively, let the number of `likes` be included when getting on accommodation